python/interpret_community/mimic/models/lightgbm_model.py

```diff
@@ -47,7 +48,9 @@ def __init__(self, function):
         self._function = function
 
     def predict_wrapper(self, X, *args, **kwargs):
-        """Wraps a prediction function from lightgbm learner to densify the input dataset.
+        """Wraps a prediction function from lightgbm learner.
+
+        If version is 3.0.0, densifies the input dataset.
```

Comment:
Nit: might want to specify the direction >= like you do below

Reply:
done, added ==